### ISSUE TITLE
Accessibility - Voiceover can not read messages in dialog boxes [macOS]

### ISSUE THREAD

**Issue by adincebic** [microsoft/vscode#55585]
- VSCode Version: 1.25.1
- OS Version: macOS High Siera 10.13.6

Steps to Reproduce:

1. Press command + F5 to turn voiceover on.
2. Launch Visual Studio Code.
3. Press option + ctrl + m to go to the menu bar.
4. Navigate right using right arrow until you find "Code" menu item (it is the first one)
5. Use down arrow to expand the menu and press enter on "check for update" option.
If you are up to date, a dialog box saying that your version of VSCode is the latest should appear.

Expected result: We should be able to read the content of the dialog box using voiceover.

Actual result: Only sound is reproduced and no description is read by voiceover.

What I tried: I tried using VO keys (option + ctrl) and arrows to navigate within the dialog, but it only reads the content behind the dialog box.
I tried using tab key to navigate, but that option does not work too.

What I noticed while using VSCode:
Usually every dialog box that is displayed by VSCode is not accessible. The dialog box that asks whether I want to save the changes is partially accessible because I can use the tab key to navigate between "cancel", "save" and "don't save" buttons, but I am still not able to read the message.

Accessible dialog boxes: You can open the "About VSCode" dialog and see that voiceover is able to read everything within that dialog.

Does this issue occur when all extensions are disabled?: Yes/No
Yes

**Comment by isidorn** [comment]
Thanks for reporting this issue. I can reproduce this behavior that voice over does not read the text in dialog boxes.
This seems to be an electron issue. Thus assigning to @Tyriar and @bpasero

**Comment by bpasero** [comment]
@isidorn can you please report this to Electron with a simple example

**Comment by isidorn** [comment]
@deepak1556 also assigning to you since I know you have this on the radar.

**Comment by isidorn** [comment]
This is fixed in Electron 11. Thus closing this.
In case some users still hit this with latest VS Code please let us know.
